muxink/src/fragmented.rs

```diff
@@ -1,8 +1,8 @@
 //! Splits frames into fragments.
 //!
 //! The wire format for fragments is `NCCC...` where `CCC...` is the data fragment and `N` is the
-//! continuation byte, which is `0x00` if more fragments are following, `0xFF` if this is the frame's
-//! last fragment.
+//! continuation byte, which is `0x00` if more fragments are following, `0xFF` if this is the
+//! frame's last fragment.
```

Comment:
Does the formatter complain if this is not altered?

Reply:
The formatter itself did the change.